buildSrc/src/main/java/Dependencies.kt

```diff
@@ -64,6 +64,7 @@ object Versions {
     const val robolectric = "4.2"
 
     const val google_ads_id_version = "16.0.0"
+    const val androidx_fragment_testing_version = "1.2.0-alpha02"
```

Comment:
Shouldn't we be using `androidx_testing` from above instead?

Reply:
That refers to a different artifact